packages/json-schema/README.md

```diff
@@ -0,0 +1,55 @@
+# @loopback/json-schema
+
+Convert a TypeScript class/model to a JSON Schema for users, leveraging LoopBack4's decorators, metadata, and reflection system.
+
+## Overview
+
+This package provides modules to easily convert LoopBack4 models that have been decorated with `@model` and `@property` to a matching JSON Schema Definition.
```

Comment:
Since `@model` and `@property` decorators are coming from `@loopback/repository` package and are technically optional, would it make sense to rename `@loopback/json-schema` to something like `@loopback/repository-json-schema` or `@loopback/model-json-schema`?

Reply:
I changed the name to `@loopback/repository-json-schema`